### DISCUSSION TITLE
using TAD coordinates with scenic wrapper function?

### DISCUSSION THREAD

**Discussion by sid5427** [aertslab/scenicplus#141] [Q&A]
Hi Seppe,

As the title suggests - is there a direct way to use the "run_scenic" wrapper function with TAD coordinates?
As opposed to following the second "step by step" tutorial and using the "get_search_space" function to incorporate the coordinates and then eventually go on to run the scenic function.

This would be useful as I can quickly drop in a pyranges object with TAD coordinates into my previous scenic code (which is based on the first tutorial).

Thanks
Sid

**Comment by SeppeDeWinter** [comment]
Hi @sid5427 

You can add the TAD coordinates (you previously generated) to this field in the scenicplus object 

```python

scplus_obj.uns['search_space'] = <TAD>

```

After this you can just run the wrapper function, it won't overwrite the custom search space you put in.


I hope this helps.

Best,

Seppe
